### ISSUE TITLE
Link examples to live editor

### ISSUE THREAD

**Issue by sidharthv96** [mermaid-js/mermaid#2473]
**Is your feature request related to a problem? Please describe.**
It's not easy to play around with example code.

**Describe the solution you'd like**
Once #2449 is complete, we could add a button on the examples to open in live editor.

cc @Yash-Singh1

**Comment by Yash-Singh1** [comment]
Should we implement this in #2449?

**Comment by sidharthv96** [comment]
Don't wanna bloat that PR too much. I guess it's easier to review and discuss on a separate PR. 

You can base the new PR off of the existing branch.
